server/model/tweet.js

```diff
@@ -1,5 +1,6 @@
-import {Record} from 'immutable';
+// @flow
+export type Tweet = {|
+  html: string;
+|}
 
-export const Tweet = Record({
-  html: null
-});
+export function createTweet(data: Tweet) { return (data: Tweet); }
```

Comment:
I think I prefer this `createX` naming – I feel like it involves a bit less cognitive overhead to figure out what's doing what. I'm sure I could get used to either style, though, as long as we're consistent.